### ISSUE TITLE
base common features

### ISSUE THREAD

**Issue by vmlf01** [porumacausa/need-and-give-docs#2]
To get discussion started, I think all these projects need these features:
- user registration
  - needer and giver equal (can be both at same time), no approval necessary
  - needer and giver have different roles (needer has to apply and be reviewed before joining)
- user profile
- item categories/tag system
- item listings
- transaction rating system
- comments/messages on item listings
- search by geo-location & text

**Comment by LuisCamacho** [comment]
I think we should add the ability to create communities.

**Comment by vmlf01** [comment]
Like user groups? Is that what you mean?

**Comment by LuisCamacho** [comment]
Yes, something like that. I think this would be a good way to organize the
site.

Groups can be small like a group of neighbors that share the lawnmower or a
big organization.

**Comment by hmmultimedia** [comment]
I don't remember of anything that misses. Good work.
maybe for social funding , is needed a payment system.
